sdk/communication/azure-communication-sms/src/samples/java/com/azure/communication/sms/samples/quickstart/ReadmeSamples.java

```diff
@@ -111,4 +148,48 @@ public void sendSMSTroubleshooting() {
             System.out.println(ex.getMessage());
         }
     }
+
+    public void catchHttpErrorOnRequestAsync() {
+        SmsAsyncClient smsClient = createSmsAsyncClientUsingAzureKeyCredential();
+        try {
+            Mono<SmsSendResult> sendResult = smsClient.send(
+                "<from-phone-number>",
+                "<to-phone-number>",
+                "Hi"
+            );
+        } catch (RuntimeException ex) {
+            System.out.println(ex.getMessage());
+        }
+    }
+
+    public void failedMessages() {
```

Comment:
Let's rename to something like "TroubleshootingSendMessageErrors"? 
Anyone have better suggestions?